pkg/service/test/util.go

```diff
@@ -237,3 +238,31 @@ func Contains(elems []string, v string) bool {
 	}
 	return false
 }
+
+// Filter the mocks based on req and res timestamp of test
+func filterTcsMocks(tc *models.TestCase, m []*models.Mock) ([]*models.Mock, error) {
+	filteredMocks := make([]*models.Mock, 0)
+
+	if tc.HttpReq.Timestamp == (time.Time{}) {
+		return filteredMocks, fmt.Errorf("request timestamp is missing of %s", tc.Name)
```

Comment:
This is not what I meant by the previous comment. We need to maintain backward compatibility . I mean even if we try with mocks and tests which doesn't have timestamps it should work like earlier. Like if tc doesn't have timestamp put all the mocks into filtered mocks and log a warning msg that your tests and mocks doesn't have timestamps please record them again etc.